bindings/python/README.md

```diff
@@ -130,4 +121,757 @@ def main():
+| [nodes]               | <code>list[str]</code>                       | <code>undefined</code> | An array of node URLs                                                                                                                                             |
+| [node_sync_interval]  | <code>int</code>                             | <code>undefined</code> | The interval for the node syncing process                                                                                                                         |
+| [node_sync_disabled]  | <code>bool</code>                            | <code>undefined</code> | Disables the node syncing process. Every node will be considered healthy and ready to use                                                                         |
+| [node_pool_urls]      | <code>str</code>                             | <code>undefined</code> | An array of node pool URLs                                                                                                                                        |
+| [request_timeout]     | <code>int</code>                             | <code>undefined</code> | Sets the default HTTP request timeout                                                                                                                             |
+| [api_timeout]         | <code>dict</code>                            | <code>undefined</code> | The API to set the request timeout. Key: 'GetHealth', 'GetInfo', 'GetPeers', 'GetTips', 'PostMessage', 'GetOutput', 'GetMilestone' Value: timeout in milliseconds |
+| [local_pow]           | <code>bool</code>                            | <code>undefined</code> | Flag determining if PoW should be done locally or remotely                                                                                                        |
+| [tips_interval]       | <code>int</code>                             | <code>undefined</code> | Time between requests for new tips during PoW                                                                                                                     |
+| [mqtt_broker_options] | <code>[BrokerOptions](#brokeroptions)</code> | <code>undefined</code> | Sets the options for the MQTT connection with the node                                                                                                            |
+
+**Returns** The constructed [Client](#client).
+
+### Full Node APIs
+
+#### get_health(): bool
+
+Gets the node health status.
+
+**Returns** whether the node is healthy.
+
+#### get_info(): InfoResponse
+
+Gets information about the node.
+
+**Returns** the [InfoResponse](#inforesponse).
+
+#### get_peers(): list[PeerDto]
+
+Gets peers of the node.
+
+**Returns** the list of [PeerDto](#peerdto).
+
+#### get_tips(): list[str]
+
+Gets two non-lazy tips.
```

Comment:
Remove `two` (also in the line below), because the amount of tips is not fixed to two

Reply:
Got it. Modified. Thanks for identifying this.